tests/functional/pv/pvc_snapshot/test_restore_snapshot_when_parent_pvc_deleted.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import green_squad
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+    polarion_id,
+    skipif_ocp_version,
+    bugzilla,
+)
+from ocs_ci.ocs.resources import pod
+from ocs_ci.helpers import helpers
+
+log = logging.getLogger(__name__)
+
+
+@green_squad
+@tier1
+@skipif_ocs_version("<4.16")
```

Comment:
Isn't the test case supported in lower versions ?

Reply:
I see the [bug2264900](url) fixed in 4.16, so I believe it is not supported in lower version.